custom_components/moonraker/sensor.py

```diff
@@ -201,10 +201,50 @@ class MoonrakerSensorDescription(SensorEntityDescription):
 async def async_setup_entry(hass, entry, async_add_entities):
     """Setup sensor platform."""
     coordinator = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities([MoonrakerSensor(coordinator, entry, desc) for desc in SENSORS])
+
+    object_list = await coordinator.async_get_printer_object_list()
+    opt_sensors = await async_setup_optional_temp_sensors(object_list)
```

Comment:
opt_sensors -> `opt_temp_sensors`

Reply:
or rename method for async_setup_optional_sensors

(I think this is better and makes the heated bed change easier)